spec.mediawiki

```diff
@@ -618,32 +840,32 @@ The client may specify the ''time in force'' of a limit order as one of: (a)
 such, the ''immediate'' option is intended for limit orders with a price that 
 crosses the spread (i.e. a taker rather than a maker). The  
 <code>ordersize</code> must be an integer multiple of the asset's 
-[[#Asset_Variables|lot size]].
+[[#Exchange_Variables|lot size]].
 
-'''JSON-RPC limit order parameters'''
+'''JSON-RPC method:''' <code>limit</code>, '''originator:''' client
 
 {|
 ! field       !! type   !! description
 |-
-| accountid   || string || client account identifier
+| accountid   || string || client's account ID
 |-
 | timestamp   || int    || UTC timestamp generated at the time of order submission of trades requires a fair amount of messaging between the 2 clients, via the DEX trade API.n
 |-
 | tradepair   || string || market. trading pair. e.g. DCR/BTC
 |-
 | ordertype   || string || "limit" for a limit order
 |-
-| direction   || string || one of "buy" or "sell"
+| side        || string || one of "buy" or "sell"
 |-
-| ordersize   || int    || order size, in atoms (*1e8)
+| ordersize   || int    || order size (atoms)
 |-
-| price       || int    || price, in atoms/Satoshi or similar (*1e8)
+| rate        || int    || price rate (atoms per unit counter-asset)
 |-
 | timeinforce || string || one of ''standing'' or ''immediate''
 |-
 | utxos       ||  &#91;[[#UTXO_Preparation|UTXO object]]&#93; || list of backing UTXOs
 |-
-| nonce       || string || a unique value which identifies this order in communications
```

Comment:
`nonce` appears to have snuck in during early roughing. It's not in the proposal or the blog and I don't believe it was ever discussed. As described, it was unnecessary because the order ID is already assumed unique.  removed here and 4 other structures or serializations.